docs/Developer Documentation/Logger/index.md

```diff
@@ -0,0 +1,159 @@
+# SDL Logger
+
+Open source version of SDL is using log4cxx for logging.
+Because of lack of log4cxx stability OEM should have an ability to replace log4cxx logger with any other logging boost or syslog.
+
+
+SDL implements Logger abstration for easily replacement of logging library.
```

Comment:
Please change to:
SDL implements Logger abstraction allowing for easy replacement of the logging library.

Reply:
Fixed with  
aeadebe